recipes/bloaty/all/test_package/conanfile.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+from conans import ConanFile, tools
+
+
+class TestPackageConan(ConanFile):
+    settings = "os"
+
+    def test(self):
+        if tools.cross_building(self.settings):
+            return
+        bloaty_path = os.path.join(self.deps_cpp_info["bloaty"].bin_paths[0], "bloaty")
```

Comment:
Will this work on Windows where the executable has a `.exe` suffix?

Reply:
Simplified